### DISCUSSION TITLE
DeepSpeech and discord

### DISCUSSION THREAD

**Discussion by aimjel** [coqui-ai/STT#1797] [Q&A]
```
    modelStream = englishModel.createStream();


    stream.on('data', chunk => {

        modelStream.feedAudioContent(chunk);
    }).on('close', () => {

        console.log('result:', modelStream.finishStream());

    });
```

When I try using a discord stream which is 48khz PCM stereo 16bit little-e with deep speech no word is recognize. I've tried transforming the stream in wav/16khz with no luck

**Comment by reuben** [comment]
One good debugging strategy is saving the stream to a file so you can listen to it and make sure nothing in your pipline is messing up the data.
